src/api/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 202302224114624) do
+ActiveRecord::Schema[7.0].define(version: 2023_04_13_080503) do
```

Comment:
```suggestion
ActiveRecord::Schema[7.0].define(version: 20230413080503) do
```

Reply:
The version format looks wrong, or does it work with the underscores? @saraycp

Reply:
In fact, I did set `20230413080503` but running `rails db:migrate:with_data` afterwards changed it to `2023_04_13_080503` (even if it didn't migrate anything).

You can see here it's a correct format: https://api.rubyonrails.org/classes/ActiveRecord/Schema/Definition/ClassMethods.html#method-i-define